src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import * as Pages from './pages';
+import { Header } from './layout';
+
+import './App.css';
+
+function App(){
+
+    return(
+        <>
+        <Header />
+        <Switch>
+            <Route exact path='/' component={Pages.Home} />
+            <Route path='/user' component={Pages.User} />
+        </Switch>
+        </>
+    );
+
+};
+
```

Comment:
Nice that you used the routing. Having 2 pages makes a huge difference on how the app scale feels like.